core/templates/dev/head/pages/exploration_editor/editor_tab/ExplorationEditorTab.js

```diff
@@ -223,9 +252,7 @@ oppia.directive('trainingPanel', [
 
           $scope.beginAddingNewResponse = function() {
             var contentId = GenerateContentIdService.getNextId(
-              StateContentIdsToAudioTranslationsService.displayed
-                .getAllContentId(),
-              COMPONENT_NAME_FEEDBACK);
+              _getExistingOutcomeContentIds(), COMPONENT_NAME_FEEDBACK);
```

Comment:
This seems to be a change in functionality -- it used to get all IDs. Is this intentional?

Reply:
Yep! the generator service use to take all the contents to look for all of the component related ids now I'm providing it with such, so there won't be any issue.